3001-4000/LIT3529Daniel.xml

```diff
@@ -37,6 +33,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          </xi:include>
       </encodingDesc>
       <profileDesc>
+         <abstract>
+            <p>Biblical Book of Daniel, including the so-called deuterocanonical books with the stories about Susanna, Bel and the dragon, 
+               the Prayer of Azariah and the Three Holy Children.</p>
```

Comment:
whole-part relations could be added here or in the corresponding records

Reply:
How can I refer to the other ID in relations? I cannot find a relation called "whole-part". I could probably use "saws:contains", but that would require defining the deuterocanonical books as independent IDs. Or do you think, it is sufficient to define LIT1259Daniel to be a part of LIT3529Daniel. This could be misleading, because as far as I know the deuterocanonical books are later additions to the original book of Daniel.

Reply:
of course the deuterocanonical books must - and already have had - their own IDs, since actually the two Daniel version have always been disambiguated through the title itself 

LIT3529Daniel (with the `title  `"Daniel, Book of (includes Daniel,  Susanna and Bel and the Dragon)" was initially created as the container for

LIT1259Daniel
LIT3194Susanna 
LIT3195Bel
LIT2250salota = LIT1828Mahale#ThreeYouths1 

I checked and they seem to already have the relation in place, but can be re-checked


There are numerous whole-part relations for various degrees of certainty
https://betamasaheft.eu/Guidelines/?id=relations

whole to part 
saws:contains can be used if you are sure that a textual unit **always** contains another textual unit (the latter may circulate also independently). 

part to whole 
`saws:formsPartOf` is used to declare that a defined Textual Unit is `always `part of another one
`ecrm:CLP46i_may_form_part_of` can be used for texts which are `not always` included in another one (and may also circulate independently)
`dcterms:isPartOf` - the more generic relation for parts of larger collections

Reply:
Yes, I think LIT2271salota should also be added among the relations here.
LIT4300PrayerAzariah has only been used once in BDLaethd3 to encode "Prayer of Azariah and the Song of the Three Holy Children...corresponds with LXX Daniel 3:28-88" (= LIT2250salota+LIT2271salota+LIT1566hababa). I would propose to delete it, as it is confusing matters unnecessarily.